src/cogent3/core/alignment.py

```diff
@@ -2285,6 +2298,103 @@ def get_features(
                 # passing self only used when self is an Alignment
                 yield seq.make_feature(feature, self)
 
+    def distance_matrix(self, calc="jaccard", k: int = 7):
+        """estimated pairwise distance between sequences.
+
+        Parameters
+        ----------
+        calc : str | Callable[[set, set], float]
+            str
+                The distance calculation method to use, by default "jaccard".
+            Callable[[set, set], float]
+                A distance calculator instance. Must compare sets of kmers and return float.
+        k : int, optional
+            The length of kmers to use in the distance calculation, by default 7.
+
+        Returns
+        -------
+        DistanceMatrix
+            Pairwise distances between sequences in the collection
+        """
+
+        from cogent3.evolve.fast_distance import DistanceMatrix
```

Comment:
I'm recommending we move distance related calculations into `cogent3.app.dist` to satisfy the "separation of concerns" principle, in terms of module organisation. In which case, this method just creates instances of the required apps using `cogent3.get_app()` (`cogent3` already imported above).